src/components/OntologyBrowser/index.tsx

```diff
@@ -27,6 +27,15 @@ type ModalState = {
   error?: Error | null;
 };
 
+const desactivateAllSameTerms = (allSameTerms: string[], transferItems: TransferItem[]) =>
+  transferItems.map((t) => {
+    if (allSameTerms.includes(t.key)) {
+      return Object.assign(t, { disabled: true });
```

Comment:
use spread operator vs Object.assign (since it s the way we do elsewhere in the portal codebase)